lib/groonga-query-log/command/format-regression-test-logs.rb

```diff
@@ -28,7 +28,12 @@
 module GroongaQueryLog
     module Command
       class FormatRegressionTestLogs
-        def initialize
+        def initialize(output = nil)
+          if output
+            @output = output
+          else
+            @output = $stdout
+          end
```

Comment:
Could you use `options={}` instead of `output=nil`? Optional parameters may be added later.

```ruby
def initialize(options={})
  @output = options[:output] || $stdout
end
```